VirtoCommerce.Platform.Tests/Assets/BlobStorageProviderTests.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.IO;
+using VirtoCommerce.Platform.Data.Assets;
+using Xunit;
+
+namespace VirtoCommerce.Platform.Tests.Assets
+{
+    public class BlobStorageProviderTests
```

Comment:
Please add `[Trait("Category", "Unit")]` attribute to test class, this attribute tells our CI to run it for each build.